src/components/controls.tsx

```diff
@@ -173,11 +173,11 @@ export class Controls extends BaseComponent<IProps, IState> {
                 <ControlLabel>Column Height (km)</ControlLabel>
                 <HorizontalContainer>
                   <RangeControl
-                    min={.1}
+                    min={.5}
                     max={25}
                     value={stagingColHeight / 1000}
                     step={.1}
-                    tickArray={[.1, 5, 10, 15, 20, 25]}
+                    tickArray={[.5, 5, 10, 15, 20, 25]}
```

Comment:
There might be a couple of related changes if the new min val of the col height slider is .5.  

Could we also change the min val of the widget to .5 here?
https://github.com/concord-consortium/geocode/blob/master/src/components/column-height-widget.tsx#L55

Not sure if it also has an impact on the VEI related col height val.  I'm assuming no, but worth a look:
https://github.com/concord-consortium/geocode/blob/master/src/utilities/vei.ts#L9

Reply:
:+1: